install_config/cluster_metrics.adoc

```diff
@@ -753,3 +753,292 @@ by performing the following steps:
 $ ansible-playbook <OPENSHIFT_ANSIBLE_DIR>/byo/openshift-cluster/openshift-metrics.yml \
    -e openshift_metrics_install_metrics=False
 ----
+
+[[openshift-prometheus]]
+== Prometheus on OpenShift
+
+Prometheus is a stand-alone, open source systems monitoring and alerting
+toolkit. You can use Prometheus to visualize metrics and alerts for OpenShift
+system resources.
+
+[IMPORTANT]
+====
+The Prometheus is a Technology Preview feature only.
+ifdef::openshift-enterprise[]
+Technology Preview features are not supported with Red Hat production service
+level agreements (SLAs), might not be functionally complete, and Red Hat does
+not recommend to use them for production. These features provide early access to
+upcoming product features, enabling customers to test functionality and provide
+feedback during the development process.
+
+For more information on Red Hat Technology Preview features support scope, see
+https://access.redhat.com/support/offerings/techpreview/.
+endif::[]
+====
+
+[[openshift-prometheus-roles]]
+=== Setting Prometheus Role Variables
+
+The Prometheus role creates:
+
+* OpenShift-Prometheus namespace.
+* Prometheus clusterrolebinding and service account.
+* PVs for Prometheus, alertmanager and alertbuffer.
+* Prometheus pod with Prometheus behind oauth-proxy, alertmanager and alert-buffer
+behind oauth-proxy.
+* Prometheus and alertmanager configmaps.
```

Comment:
alertmanager configmap is named `prometheus-alerts`

Reply:
Should I change all references to alertmanager to prometheus-alerts? Or is does prometheus-alerts apply only to the alertmanager configmap?

Reply:
only configmap